tensorrt_llm/_torch/visual_gen/models/wan/pipeline_wan_i2v.py

```
@@ -363,14 +363,45 @@ def _run_warmup(self, height: int, width: int, num_frames: int, steps: int) -> N
             )
 
+    DEFAULT_GENERATION_PARAMS = {
+        "height": 480,
+        "width": 832,
+        "num_inference_steps": 50,
+        "guidance_scale": 5.0,
+        "max_sequence_length": 512,
+        "num_frames": 81,
+        "frame_rate": 24.0,
+        "image_cond_strength": 1.0,
+    }
```

Ruff (0.15.9)

[warning] Mutable default value for class attribute

(RUF012)